### PR TITLE
Small claim fixes

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Fixing issues I tagged on https://github.com/gnosis/cowswap/pull/2142#pullrequestreview-852325822

1. USDC for rinkeby, so we can test USDC claiming :)
2. Switched from WETH to ETH (and xDAI)
3. Removed unused vars
4. Small refactor in a hook

Screen Shot 2022-01-14 at 08 57 31
Screen Shot 2022-01-14 at 08 57 45

  # To Test

1. Main thing is checking the approval flow for USDC and ETH
* Now you'll see your USDC balance
* Now you'll see ETH instead of WETH

# Accounts

Pick one of these to see all of that

-  0xD5e900280Eb1aDe4E583c2bF2414be247E298435
-  0x7946Fce94A2350a1076a8F97105262c9E07e2a9d
-  0xd46C95581Ac035Bc10D80DD2a121D444e63C2E24
-  0xe93BDEe77cea1aFa9c891C78dca51A16fA287630
-  0x0280a0D231bc48B6A5EcEc3E1dc653Aae6abFf29
-  0x7E5966fFfa6092c6A9b4685A61969C44fa77C18A
-  0xA624B88Bb6A2e752fbe590266a56B940D0206cA4
-  0xa6Caab2e5b5cB306af5ee06a3E4d3b56d9D3B73e
-  0xd0ef9fD6b5795DF8add90E1dc0A16D66e2C5C1E6
-  0x2C8CbbbfEB7E116Cc27e7Ac6cC44958427D0bBA2
-  0xb04d950C9146BAd8263677A2A8DD0Ebd096Fab17
-  0xcc32131FdB8345996402e816B91A85cDf033f017
-  0xE568be91c612302E049F3A7c52Aec76E77E8DBB6
-  0xF507BD42B239E805d4Fa5551D82577D9113830a3
-  0xAbc20633c1c375854251daec0b13033da5D3949d
-  0x34FB768EB3941F308a08526C7d9ec4587291E26a
-  0x69EB8bcEf34c2Fac01ae054B1A77E6b3689e4730
-  0xB15bFBF5E4A111f1dC54546D35221065F12CC285
-  0x912b14277279156dd1f2501B0900982fA04f02D1
-  0xC387532B652C0FeA259Ed4272754c5C0F969aA24

### CODE REVIEW

File: src/custom/state/claim/hooks/utils.ts [hunk 1]
```diff
@@ -161,17 +164,18 @@ export type PaidClaimTypeToPriceMap = {
...
- *
- * @param type
  */
 export function claimTypeToTokenAmount(type: ClaimType, chainId: SupportedChainId) {
   switch (type) {
     case ClaimType.GnoOption:
       return { token: GNO[chainId], amount: GNO_PRICE }
-    case ClaimType.Investor:
-      return { token: USDC, amount: USDC_PRICE }
+    case ClaimType.Investor: {
```
Comment (W3stside) [inline src/custom/state/claim/hooks/utils.ts:172]: this will probably change with my USDC approve pr #2154 as i created a USDC map, tho maybe not idk

**Review by W3stside — APPROVED** [review]

File: src/custom/state/claim/hooks/utils.ts (hunk 1 above)
Comment (alfetopito) [reply src/custom/state/claim/hooks/utils.ts:172]: Yep, that is another way of fixing it

Comment (alfetopito) [reply src/custom/state/claim/hooks/utils.ts:172]: Would you rather I remove it then?

Comment (alfetopito) [reply src/custom/state/claim/hooks/utils.ts:172]: Yours is definitely more complete as I missed the USDC_XDAI. 

Should use that instead. Can you apply the fix in your PR? I'll remove this part from mine

Comment (alfetopito) [reply src/custom/state/claim/hooks/utils.ts:172]: Ok ok, as agreed I'll adjust after https://github.com/gnosis/cowswap/pull/2154 is merged

**Review by anxolin — COMMENTED** [review]
APPROVE

Regarding `GpEther`, for now on, i think we should use `Cow` instead of `Gp` for prefixes/suffixes. Although i wouldn't care too much about fixing the old ones in the code (unless we touch there and we fee like it)

**Comment by elena-zh** [comment]
LGTM!

**Pull request merged**